src/libraries/System.Private.DataContractSerialization/src/System/Runtime/Serialization/Json/JsonCollectionDataContract.cs

```diff
@@ -147,18 +150,19 @@ public override object ReadJsonValueCore(XmlReaderDelegator jsonReader, XmlObjec
             return o;
         }
 
-        public override void WriteJsonValueCore(XmlWriterDelegator jsonWriter, object obj, XmlObjectSerializerWriteContextComplexJson context, RuntimeTypeHandle declaredTypeHandle)
+        public override void WriteJsonValueCore(XmlWriterDelegator jsonWriter, object obj, XmlObjectSerializerWriteContextComplexJson? context, RuntimeTypeHandle declaredTypeHandle)
```

Comment:
should obj be nullable?

Reply:
https://github.com/dotnet/runtime/pull/41476#discussion_r479391298